packages/expo-module-template/ios/ExpoModuleTemplate.podspec

```diff
@@ -13,8 +13,8 @@ Pod::Spec.new do |s|
   s.platform       = :ios, '12.0'
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
-  s.source_files   = 'EXModuleTemplate/**/*.{h,m}'
-  s.preserve_paths = 'EXModuleTemplate/**/*.{h,m}'
+  s.source_files   = 'ExpoModuleTemplate/**/*.{h,m,swift}'
+  s.preserve_paths = 'ExpoModuleTemplate/**/*.{h,m,swift}'
```

Comment:
In the new modules I've stopped using these additional directory — it's just useless 🙂 
```suggestion
  s.source_files   = '**/*.{h,m,swift}'
  s.preserve_paths = '**/*.{h,m,swift}'
```